INSTALL_FROM_GITHUB.md

```diff
@@ -43,7 +43,46 @@ Once the success message appears you are done.
 #### Option 2 - With the command line Project Generator
 * from the OF root directory, run `projectGenerator -r -o"." examples`
 
-## How to submit your pull requests
+
+
+# How to update openFrameworks from github
+
+It's likely that at some point you will want to update your local openFrameworks to include new changes available in github. To update openFrameworks, we first need to figure out if you have several remotes already configured. Run
+
+    git remote -v
+    
+And study the output. It may look something like this:
+
+```
+origin	git@github.com:your-user-name/openFrameworks.git (fetch)
+origin	git@github.com:your-user-name/openFrameworks.git (push)
+upstream	https://github.com/openframeworks/openFrameworks.git (fetch)
+upstream	https://github.com/openframeworks/openFrameworks.git (push)
+```
+
+This means that there are two remotes:
+- the first two lines point at your own fork in github
+- the next two lines point at the official openFrameworks repository
+
+It is possible that you only have one or the other. If you are missing the official remote you can add it like this:
+
+    git remote add upstream git@github.com:openframeworks/openFrameworks.git
+
+Note that `upstream` is a name you choose, and it's how you will refer to the official openframeworks repository. Use that same name in the following command to update your local copy of openFrameworks:
+
+```bash
+git pull upstream master
+git submodule update --recursive
```

Comment:
I ran into this recently. 
If you haven't pulled in the submodules you have to do this first: 
`git submodule update --init --recursive`

Reply:
Added :)